openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -243,41 +243,8 @@ Requirements for both the Wallet and the Verifier:
   * ISO mdoc Credential Format specific DCQL parameters as defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be used.
   * Verifier MAY request more than one Credential in the same request.
   * When multiple ISO mdocs are being returned, each ISO mdoc MUST be returned in a separate `DeviceResponse` (as defined in 8.3.2.1.2.2 of [@!ISO.18013-5]), each matching to a respective DCQL query. Therefore, the resulting `vp_token` contains multiple `DeviceResponse` instances.
-
-### Session Transcript
-
-The `SessionTranscript` as defined in Section 9.1.5.1 in [@ISO.18013-5] MUST be used with the following changes:
-
-* `DeviceEngagementBytes` MUST be null.
-* `EReaderKeyBytes` MUST be null.
-
-```
-SessionTranscript = [
-  null,
-  null,
-  Handover
-]
-```
-
-The `Handover` element is defined as following:
-
-```
-Handover = OID4VPDCAPIHandover
-OID4VPDCAPIHandover = [
-  "OID4VPDCAPIHandover"
-  clientId
-  origin
-  nonce
-]
-
-clientId = tstr ; using UTF-8
-origin = tstr ; using UTF-8
-nonce = tstr ; using UTF-8
-```
-
-* "OID4VPDCAPIHandover" is a string that identifies the type of handover structure as a security measure to prevent confusion of handovers.
-* `clientId` and `nonce` parameters in the Handover MUST be the `client_id` and `nonce` parameters included in the Authorization Request from the Verifier.
-* `origin` in the Handover is the origin of the Verifer, obtained from the Web-platform and/or app platform.
+  * The `SessionTranscript` and `Handover` CBOR structures MUST be generated in accordance with Annex B.3.4.1 of [@!OIDF.OID4VP].  
+  * The hash function used to compute the `OID4VPDCAPIHandoverInfoHash` value in the `OID4VPDCAPIHandover` CBOR structure, as defined in Annex B.3.4.1, MUST be one of the following: SHA-256, SHA-384, SHA-512.
```

Comment:
Not entirely sure about this one, but we are currently only talking about  sha-256 in HAIP (currently scoped to sd-jwt vc - https://openid.github.io/oid4vc-haip/openid4vc-high-assurance-interoperability-profile-wg-draft.html#section-9).

Would it make sense to remove that statement about hash functions here and move that into the general section profiling hash functions (perhaps with another PR, not this one)?

Reply:
I updated the OID4VP PR to allow only SHA-256 since `SessionTranscript` is not transmitted. I updated this PR to reflect this change. Please review again @c2bo @timcappalli @hlozi @leecam @jogu .